crates/red_knot_wasm/src/lib.rs

```diff
@@ -354,7 +354,9 @@ impl Diagnostic {
 
     #[wasm_bindgen]
     pub fn display(&self, workspace: &Workspace) -> JsString {
-        let config = DisplayDiagnosticConfig::default().color(false);
+        let config = DisplayDiagnosticConfig::default()
+            .color(false)
+            .format(DiagnosticFormat::Concise);
```

Comment:
The idea of `display` is to render the full diagnostic. So I think this should remain `::default` (This is the main difference to `message`)

Reply:
Without this, the message in hover shows only "Reveal type"

Reply:
Are you sure, because this method isn't used in the playground (and on hover uses types, not diagnostics)

Reply:
Oh, you're correct. I must've opened the playground from `main` and didn't notice it's not from localhost